macaroonbakery/utils.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import json
+import six
+import webbrowser
+
+from pymacaroons import Macaroon
+
+from macaroonbakery import json_serializer
+
+
+def deserialize(json_macaroon):
+    '''Deserialize a JSON macaroon into a macaroon object from pymacaroons.
+
+    @param the JSON macaroon to deserialize as a dict.
+    @return the deserialized macaroon object.
+    '''
+    return Macaroon.deserialize(json.dumps(json_macaroon),
+                                json_serializer.JsonSerializer())
+
+
+def serialize_macaroon_string(macaroon):
+    '''Serialize macaroon object to string.
+
+    @param macaroon object to be serialized.
+    @return a string serialization form of the macaroon.
+    '''
+    return macaroon.serialize(json_serializer.JsonSerializer())
+
+
+def add_padding(b):
```

Comment:
add_base64_padding ?
(the utils module isn't really about base64, so a more specific name seems better to me)